src/main/scala/mesosphere/marathon/core/health/impl/HealthCheckWorkerActor.scala

```diff
@@ -87,37 +66,6 @@ class HealthCheckWorkerActor(implicit mat: Materializer) extends Actor with Stri
     }
   }
 
-  def http(
```

Comment:
HTTP/S health checks were moved to [MarathonHttpHealthCheckActor](https://github.com/mesosphere/marathon/pull/6840/files#diff-22c91a4d5e8e772bc8f7bde87338f615R314)